### ISSUE TITLE
Code coverage not working

### ISSUE THREAD

**Issue by alepez** [intellij-rust/intellij-rust#4968]
## Environment

* **IntelliJ Rust plugin version:** 0.2.115.2157-193
* **Rust toolchain version:** rustc 1.41.0-nightly (710a362dc 2019-12-05)
* **IDE name and version:** CLion 2019.3.4
* **Operating system:** Linux Arch
* **gcov**: gcov (Arch Linux 9.2.1+20200130-2) 9.2.1 20200130
* **lcov**: LCOV version 1.14-6-g40580cd
* **grcov**: grcov 0.5.5

## Problem description

Code coverage does not work

## Steps to reproduce

Create a project with two files:

Cargo.toml

```toml
[package]
name = "rust-coverage-clion"
version = "0.1.0"
edition = "2018"

[dependencies]
```

lib.rs

```rust
fn sum(a: u32, b: u32) -> u32 {
    dbg!((a, b));
    a + b
}


#[cfg(test)]
mod tests {
    use super::*;

    #[test]
    fn it_works() {
        assert_eq!(sum(2, 2), 4);
    }
}
```

Add a run configuration "test" of type "Cargo command", where command is "test"

Click on "Run 'test' with coverage"

CLion successfully build the project and launches test. Test results are visible (one test, it succeeds) and `dbg!` output is shown as expected.

Code Coverage is not highlighted in the gutter.

Clicking "Run -> Show code coverage data" shows a window with a message "no coverage suites configured"

**Comment by dima74** [comment]
@alepez could you please check whether this was fixed in latest release 0.3.126 (which includes #5689)

**Comment by alepez** [comment]
@dima74 It works! 

Before 0.3.126:

![2020-07-23_16-32](https://user-images.githubusercontent.com/877244/88299246-71237800-cd02-11ea-863c-3859c15afb72.png)


After 0.3.126 update:

![2020-07-23_16-34](https://user-images.githubusercontent.com/877244/88299261-74b6ff00-cd02-11ea-9a74-b2cc25ca8fc5.png)

**Comment by dima74** [comment]
@alepez Great!

**Comment by mleonhard** [comment]
Code coverage was not working for me with Rust plugin 0.3.143.3737-203 in CLion.  My project uses a [cargo workspace](https://doc.rust-lang.org/cargo/reference/workspaces.html).  I got coverage working with this Run Configuration:
- Command: `test --package package1`
- Working directory: the workspace directory, not the package directory

**Comment by mleonhard** [comment]
Code coverage is not working for me with Rust plugin 0.3.143.3737-203 in CLion 2020.3.3.  My project uses a [cargo workspace](https://doc.rust-lang.org/cargo/reference/workspaces.html).  When I click the Coverage button, the tests run but the coverage screen does not appear.

I made these changes to the Run Configuration:
- Command: `test --package package1`
- Working directory: the workspace directory, not the package directory

Now the coverage screen appears, but all files show 0 lines covered.

I filed an issue with JetBrains: https://youtrack.jetbrains.com/issue/CPP-24459
